packages/mui-material/src/Select/Select.test.js

```diff
@@ -34,6 +34,54 @@ describe('<Select />', () => {
     skip: ['componentProp', 'componentsProp', 'themeVariants', 'themeStyleOverrides'],
   }));
 
+  describe('WCAG 2.5.2 - Pointer Cancellation', () => {
```

Comment:
I think this is enough

```suggestion
  describe('Pointer cancellation', () => {
```